### PR TITLE
Min value should be Decimal

### PR DESCRIPTION
**Pull request by kdelee**
This hopefully resolves error message seen in logs sometimes about "should be Decimal type"

# ISSUE TYPE
- Bug, Docs Fix or other nominal change

# COMPONENT NAME

- API

### CODE REVIEW

File: awx/main/models/ha.py [hunk 1]
```diff
@@ -160,7 +160,7 @@ def __str__(self):
         default=100,
         editable=False,
     )
-    capacity_adjustment = models.DecimalField(default=Decimal(1.0), max_digits=3, decimal_places=2, validators=[MinValueValidator(0)])
+    capacity_adjustment = models.DecimalField(default=Decimal(1.0), max_digits=3, decimal_places=2, validators=[MinValueValidator(Decimal(0.0))])
```
Comment (AlanCoding) [inline awx/main/models/ha.py:163]: Does this not require a new migration file?

Comment (kdelee) [reply awx/main/models/ha.py:163]:
```
bash-5.1$ awx-manage makemigrations
No changes detected
```
Does not appear so

Comment (kdelee) [reply awx/main/models/ha.py:163]: Don't think it actually changes the datatype, but rather validator function in Django

Comment (AlanCoding) [reply awx/main/models/ha.py:163]: validators had some weird place in both the fields and the model Meta. Weird but possible.

**Review by AlanCoding — APPROVED** [review]

**Comment by AlanCoding** [comment]
A test could be nice too.

**Pull request merged**